src/components/themed/SimpleTextInput.tsx

```diff
@@ -52,12 +52,18 @@ export interface SimpleTextInputProps extends MarginRemProps {
   secureTextEntry?: boolean // Defaults to 'false'
   testID?: string
 
-  // Unless 'autoFocus' is passed explicitly in the props, Search Bars 'autoFocus' and 'regular' text inputs don't.
+  /** Unless 'autoFocus' is passed explicitly in the props, Search Bars
+  'autoFocus' and 'regular' text inputs don't. */
   autoFocus?: boolean // Defaults to 'true'
 
-  // Unless 'blurOnClear' is passed explicitly in the props, Search Bars calls 'blur' when cleared and text inputs don't call 'blur' when cleared.
+  /** Unless 'blurOnClear' is passed explicitly in the props, Search Bars calls
+   * 'blur' when cleared and text inputs don't call 'blur' when cleared. */
   blurOnClear?: boolean // Defaults to 'false'
 
+  /** Manually control the whether the input appears selected. Visual only,
+   * mutually exclusive with the text input's true blur/focus state. */
+  selected?: boolean
```

Comment:
I thought were were going with `active`? The problem with `selected` is that it is used as a word for selector input types, so it seems misplaced here.

Reply:
I remember we agreed the other way around... Don't think selector input types was the rationale but I can't think of it now so I'll just change it.